crates/storage-api/src/outbox_table/mod.rs

```diff
@@ -29,8 +31,8 @@ pub enum OutboxMessage {
         response: ResponseResult,
     },
 
-    /// Kill command to send to another partition processor
-    Kill(FullInvocationId),
+    /// Terminate invocation to send to another partition processor
+    TerminateInvocation(InvocationTermination),
```

Comment:
Maybe just `Termination` or `InvocationTermination` to be consistent with above variants?

Reply:
Will rename it into `InvocationTermination`.